universal-login-commons/lib/core/constants/emojiColors.ts

```diff
@@ -1,6 +1,6 @@
 export const EMOJI_COLORS = [
   '#B9C3F8',
-  '#F5FF98',
+  '#E7D300',
```

Comment:
Where is the color coming from?

Reply:
Together with Asia, we have chosen this color.